src/components/contextpopup/partials/contextpopup.html

```diff
@@ -17,7 +17,7 @@
               <td>{{coord2056}}</td>
           </tr>
           <tr>
-              <td>WGS 84 (long/lat)</td>
+              <td>WGS 84 (lat/lon)</td>
               <td>{{coord4326}}</td>
```

Comment:
I think @davidoesch wants 2 lines : one with normal coordinates and one with the `toStringHDMS` transformation.